src/site/content/en/blog/proxx-announce/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Introducing PROXX
+subhead: a game of proximity, inspired by minesweeper
+authors:
+  - kosamari
+date: 2019-05-09
+hero: hero.jpg
+alt: a logo image of PROXX.
+description: |
+  PROXX as a minesweeper-like game built as a PWA. It works on a vast variety of devices and progressively enhances the visual appearance the more capable the device is.
+tags:
+  - post
+  - PROXX
+---
+
+The team that brought you [squoosh.app](https://squoosh.app) is back! This time, we built a web-based game called PROXX ([proxx.app](https://proxx.app)). PROXX is a game of proximity inspired by the legendary game Minesweeper. The game is situated in the space and your job is to find out where the black holes are. It works on all kinds of devices — from desktop all the way to feature phones. Users can play the game using a mouse, keyboard, d-pad even with a screen reader.
+
+<figure class="w-figure w-figure--fullbleed">
+<video controls loop muted poster="https://storage.googleapis.com/webfundamentals-assets/proxx-announce-blogpost/poster.jpg">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/proxx-announce-blogpost/kaios_vp8.webm" type="video/webm; codecs=vp8">
```

Comment:
I sent Surma a link to a bucket for web.dev. If possible I'd like to see if we can use that.